closure/goog/events/eventid.js

```diff
@@ -18,15 +18,15 @@ goog.provide('goog.events.EventId');
 
 /**
  * A templated class that is used when registering for events. Typical usage:
- * <code>
- *   /** @type {goog.events.EventId<MyEventObj>}
+ * <pre>
+ *   /** @type {goog.events.EventId&lt;MyEventObj&gt;} * /
```

Comment:
If we're going to pollute the example with entities, we might as well use 	&#47; for the closing slash.